MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -10,6 +10,135 @@ keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .ne
 
 # Pull To Refresh in MAUI DataGrid (SfDataGrid)
 
+The data grid enables the `PullToRefresh` option by setting the [SfDataGrid.AllowPullToRefresh]() property to `true`, the control supports for refreshing the data source at runtime while doing the pull to refresh action. 
```

Comment:
To enable the `PullToRefresh` option in the data grid, set the [SfDataGrid.AllowPullToRefresh]() property to `true`. This allows the control to refresh the data source at runtime when the pull-to-refresh gesture is performed.

Reply:
modified